src/ripple/rpc/tests/JSONRPC.test.cpp

```diff
@@ -932,21 +932,21 @@ R"({
 { "Non-empty SigningPubKey in submit_multisigned.",
 R"({
     "command": "submit_multisigned",
-    "Signers": [
-        {
-            "Signer": {
-                "Account": "rPcNzota6B8YBokhYtcTNqQVCngtbnWfux",
-                "TxnSignature": "3045022100F9ED357606932697A4FAB2BE7F222C21DD93CA4CFDD90357AADD07465E8457D6022038173193E3DFFFB5D78DD738CC0905395F885DA65B98FDB9793901FE3FD26ECE",
-                "SigningPubKey": "02FE36A690D6973D55F88553F5D2C4202DE75F2CF8A6D0E17C70AC223F044501F8"
-            }
-        }
-    ],
     "tx_json": {
         "Account": "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
         "Amount": "1000000000",
         "Destination": "rnUy2SHTrB9DubsPmkJZUXTf5FcNDGrYEA",
         "Fee": 50,
         "Sequence": 0,
+        "Signers": [
```

Comment:
Right.  But I think the sorting is the only modification that `submit_multisigned` does.  I did that because a text sort may not give the right value.  Doing the sort in `submit_multisigned` made it a lot easier to write my python tests.